data-collection/AdminPortal/Intro.md

```diff
@@ -0,0 +1,12 @@
+# Introduction to the Admin Portal
+The Admin Portal is the place where organisations and deployments are configured. 
+
+When a healthcare institute or study program wants to use Huma for remote data collection, the Huma operations team will build a new Organisation for them within the Admin Portal.
+
+This Organisation can then have multiple Deployments associated to it, either to separate out cohorts of patients or to address different clinical applications.
+
+Each Deployment can have its own specific branding, features, onboarding steps and modules that will be made available to patients. 
+## Users
+To create a completely new Organisation configuration, you need to be invited as an Organisation Owner. You can email ithelpdesk@huma.com to request an invitation. 
+
+Once the Organisation and Deployment configurations are complete, Organisation Staff or Access Controllers can be invited to manage it. The Access Controller can then invite an Admin user to manage the Clinician Portal, which is where care and research teams can view patient data.
```

Comment:
Once the Organisation and Deployment configurations are complete, Admin or Access Controllers can be invited to the Clinician portal to manage the access.